### PR TITLE
Tagging ModernGL and MeshIO

### PR DESCRIPTION
**Pull request by SimonDanisch**
Removing Depwarns and in ModernGL getting rid of Julia 0.3

### CODE REVIEW

**Comment by ararslan** [comment]
Looks like you're only tagging MeshIO?

**Comment by SimonDanisch** [comment]
Woops... Thanks, that's wrong :P

**Comment by SimonDanisch** [comment]
Fixed!

File: ModernGL/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+Compat 0.7.15
```
Comment (tkelman) [inline ModernGL/versions/0.1.0/requires]: `@static` needs 0.7.19

Comment (tkelman) [inline ModernGL/versions/0.1.0/requires]: and `is_unix` etc needs 0.7.20 https://github.com/JuliaLang/Compat.jl/commit/85f886227c332507ec5b31627e0b3a9fbb6aa274

**Comment by SimonDanisch** [comment]
Thanks! :)

**Pull request merged**